pkgs/development/libraries/ocl-icd/default.nix

```diff
@@ -1,21 +1,24 @@
-{stdenv, fetchurl, ruby, opencl-headers, addOpenGLRunpath }:
+{ stdenv, fetchFromGitHub, addOpenGLRunpath, autoconf, automake, libtool, opencl-headers, ruby }:
 
 stdenv.mkDerivation rec {
   name = "ocl-icd-${version}";
-  version = "2.2.10";
+  version = "2.2.12";
 
-  src = fetchurl {
-    url = "https://forge.imag.fr/frs/download.php/810/${name}.tar.gz";
-    sha256 = "0f14gpa13sdm0kzqv5yycp4pschbmi6n5fj7wl4ilspzsrqcgqr2";
+  src = fetchFromGitHub {
+    owner = "OCL-dev";
+    repo = "ocl-icd";
+    # Only doc changes since release version
```

Comment:
This is not really necessary IMO.

Reply:
Also I cannot believe it if the imports of this package have changed :smile:

Reply:
Haha, I mean that there are only doc changes [between **2.2.12** and b5880e5](https://github.com/OCL-dev/ocl-icd/compare/v2.2.12...b5880e5), not between 2.2.10 (what we have now) and b5880e5.

Anyway, the import changes result from the change to GitHub's 2.2.10 rather than the update - check the individual commit diffs to see what I mean. The previous tarball was dirty - the build artifacts meant we didn't need to run the bootstrap before. Now we have to do it properly - I should've been explicit about that, sorry :)
(Apparently the tarballs are [supposed to include the autotool output](https://github.com/OCL-dev/ocl-icd/commit/fd90ac3ccdb30f5d8c1f4c6737acdb55c81b9777), but they don't!)

Reply:
How would you like to proceed?